setting_up_firely_server/deployment/azureMarketplace.rst

```diff
@@ -0,0 +1,65 @@
+.. _deploy_azure_marketplace:
+
+Using Firely Server from Azure Marketplace
+==========================================
+
+Firely Server is available in the Azure Marketplace, which allows you to deploy it easily on Azure Kubernetes Service (AKS) with a single click. 
+This deployment method uses an embedded Helm chart, which simplifies the setup process and provides a consistent configuration experience.
+The Azure Marketplace deployment is ideal for users who want to quickly get started with Firely Server on Azure without needing to manually 
+configure Kubernetes resources.
+
+In order to deploy Firely Server from the Azure Marketplace, follow these steps:
+
+1. Go to the `Azure Marketplace <https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=Firely&page=1>`_ and search for "Firely Server".
+2. Click on the "Create" button to start the deployment process.
+3. On the first panel, provide the Azure subscription, the resource group and the cluster where Firely Server should be deployed.
```

Comment:
I'd highlight that it's required to manually create the cluster beforehand? Can we link to instructions on how to do this?